### ISSUE TITLE
changing header after you filter wavelengths

### ISSUE THREAD

**Issue by fedhere** [nyusngroup/SESNspectraPCA#7]
Hi Marc, I was telling Umer that he should use your SN class and modify it for himself to include the rebinning function as a method of the class, and I was telling him that this shoud change both the eavelength, flux, and the details related to that in the header. In doing that I noticed that your wavelength filter function does not in fact modify the header. Shouldn't it tho?

Also I enabled issues that were disabled.

**Comment by mmoJazz** [comment]
Marc can you please respond to this and cc me on this?

**Comment by fedhere** [comment]
Hi, can this be closed? @marxwillia

**Comment by marxwillia** [comment]
It's a little complicated.  The SNIDsn class already keeps track of the wavelengths in an array, and these are indeed changed when the wavelength filter is applied:https://github.com/nyusngroup/SESNspectraPCA/blob/9171b4062a3334531dfb5d9aae8227fb19539b39/code/SNIDsn.py#L589

The snidSN.header refers to the header information in the loaded SNID template.  This information is currently not really used, except in the initialization of the SNIDsn object.  There is currently no support for outputting a SNID .lnw style template, but I hope to add this functionality.  In that case, it will be important to change the actual SNID template header.  I will keep this issue open as a reminder.

**Comment by fedhere** [comment]
absolutely: having the wrong info in the header is a sure way to end up
with bugs
F
___________________________________________________
federica b. bianco
*http://fbb.space/ <http://fbb.space/>*
____________________________________________________


                                     -                         _`
                                   `+*-                       ;*;
                                  `o**=                  `   :**o.
                     `.~.`        ;*ooo`                ~~  .o*oo=
                    `-----~       =oio*                ~--` :ooo**.^`
 `
      ;`            .------`      :oee;               `---^ :e*i!?oi^
+_  -io**=
      $$i.        `.:;_---`   ~`  `+=_                `---` .*ee?!i+
i: -eo*oe;       .~^
      $OOOe_      -__++_-`   ^-.   ^ ``                .~_:_.-*!!e*~
~_~i?eo*o~      `---^
      $OOOOO?;`   -----^     `-^  ^__:-`       ^~      ~;.    .*ee:
-. -!?eo**       ^----
      $OOOOOOO$i. `~-^        ^.^`^--_^        _-    ^=- `~:;;+++^  --
  _!?***:        --~`
      :_;o?OOOOO$*-      `_:-~^    ``     ~.  ~:-   ;o.:+:~`~i+++_;_
  .ie**+
           ^;!OOOOO!_   :i^               i:  --` ^*e+:.=       ~o`
^::``.`
              .=%OOOO?_o:                 ;. `..~i$o_^..^+`
`;++++___.
                 _eOOOOo^               .:ie%?!e?*_;+_oo=^__.        ^
-i*o_  `
                   ~o$OO$*:..-^  .-::=e$$!=-`    `io**eo*:  ^`
~;:~*oii~ ~::~
                     ^?OOOOOOOO%$OOOO%o:^    +_  :*o*e*o*+    `.:-
:=+~.::. `++++.
                      `;i$O?%O%=io*e?$%e=~   oo^ +*oo*oo*;
.io*i^^_--`    ^++++_     ^-~
                         _; `=Oi`     `~;o!=^~=^ _**ooo**_    ~*ioo=
^.^      ;+++_     ---`
                              ;%%-         ~o=`  `oeoo:.;~     i*oii^
       `:++-     .--
                                +$~          ..   ;*e;          _iii:
         `.~      .^
                          ~--.  ^;%=~_;++++:`     ^*_
                         ~+:--``-`^;o!*+-  `:;`    `
             `.
                        `;++_-. ^    `.-_+:~  _;~
             --^
                        ^:++_-^     ^-;_    ..``~-.
             -_-.
                        `_;;-~      .-_-       ^---``
              __--
                         --_~       `--`       ^--;_-~
              ----^
                         .-^          .         ~_+_--^
              ----
                          `                      -;---.
              ^--^
                                                 `_---~
               .`
                                                  ^---.
          ``
                                                   `~-`
         `++.
                                                     .
         ^++_   -~

          ~:-   ~~

            `

             ^.

             ^_.

              --~

              ~-^

              `^

Please consider the environment before printing this e-mail
